soh/soh/Enhancements/randomizer/3drando/fill.cpp

```diff
@@ -488,14 +488,16 @@ static void GeneratePlaythrough() {
 RandomizerArea LookForExternalArea(Area* curRegion, std::vector<RandomizerRegion> alreadyChecked){//RANDOTODO curREGION
```

Comment:
It is better to pass alreadyChecked vector as a reference. Otherwise, you are making a copy of a whole vector.
`std::vector<RandomizerRegion>& alreadyChecked`

Reply:
Also, `const Area* const currentRegion`. Const-ness is a good fullproof. Let's avoid shortenings like 'curr' also.